web/Dockerfile

```
@@ -22,12 +55,12 @@ RUN yarn install  --immutable --immutable-cache
 FROM base as development
 CMD ["yarn", "start"]
 
+FROM server AS nginx-dev

```

```suggestion
FROM server AS nginx-dev

```

Consider naming here, as there's inconsistencies between this line and line 64;
```
FROM server AS prod
```

---

The idea here, is to have similar naming convention as before (e.g. `prod`, `dev`), but since the same Dockerfile contains two services we need to distinguish between the version of node that'll be running in dev mode along with nginx.

Any suggestions on a (sensible) naming convention for containers with multistage?